README.md

```diff
@@ -17,11 +13,7 @@ All this in less than 400 lines of YAML config. View the source in the [lowdefy.
 - Clone this repository or copy the content of the `lowdefy.yaml` into a `lowdefy.yaml` file in your local folder.
 - Create a Google sheet and share the sheet with a GService API account, see [the Google sheets Lowdefy docs](https://docs.lowdefy.com/GoogleSheet) for an in depth guide on linking a Google sheet.
 - Create `.env` file and set both the `LOWDEFY_SECRET_SHEETS_CLIENT_EMAIL` and `LOWDEFY_SECRET_SHEETS_PRIVATE_KEY` environment variables. (Make sure to never commit your secrets to your code repository.)
-- In the command console, navigate to your project folder and run the Lowdefy CLI: `npx lowdefy@latest dev`.
-
-## Deploying this example
-
-This simplest solution is to deploy this example to [Netlify](https://netlify.com). See [the Netlify Deployment instructions](https://docs.lowdefy.com/deployment) for more detail on how to deploy a Lowdefy app to Netlify.
+- In the command console, navigate to your project folder and run the Lowdefy CLI: `pnpx lowdefy@4.0.0-alpha.37 dev`.
 
```

Comment:
`pnpx lowdefy@4 dev`

Reply:
Change made in 70b1bfc4fe3d0bd71c203a67b54f01188ee76c01